config/src/main/java/org/hyperledger/besu/config/QbftConfigOptions.java

```diff
@@ -20,6 +20,8 @@
 
   Optional<String> getValidatorContractAddress();
 
+  boolean hasValidatorOverrides();
```

Comment:
This doesn't really belong as part of QbftConfigOptions as it's not a possible config option for QBFT. Might need to create another type to hold this information.